app/models/user.rb

```diff
@@ -107,6 +107,10 @@ def birth_date
     Formatters::DateFormatter.format_date(birth_date)
   end
 
+  def address
+    mpi.profile&.address
+  end
```

Comment:
I wonder if this function should be expanded to always return a hash, like this:

> def address
>   address = mpi_profile&.address
>  {
>    street: address&.street,
>    city: address&.city,
>    state: address&.state,
>    country: address&.country,
>    zip: address&.zip
>  }
> end

That could make it so the other anything calling `user.address` could always assume that the fields on `address` will be there, even if it still is `nil`, and could let you do `user.address.state` without the safety operators.